iotivity/resource/examples/os_examples/client/SConscript

```diff
@@ -25,6 +25,10 @@ examples_env.AppendUnique(CPPPATH=[
 	Dir('.').srcnode().path
 ])
 
+examples_env.Append(LIBS=['devMgmt'])
+examples_env.Append(LIBPATH=['#/../model', '/usr/lib', '/usr/local/lib'])
+examples_env.Append(CPPDEFINES=['DEVMGMT_TEST_MODE_ON'])
```

Comment:
As mentioned by @BlaCkinkGJ, to run the test code, you must define "DEVMGMT_TEST_MODE_ON" at the compile time.

e.g. (on Scons) append CPPDEFINES with the value "DEVMGMT_TEST_MODE_ON".